moveit_ros/moveit_servo/src/pose_tracking.cpp

```diff
@@ -64,37 +66,37 @@ PoseTracking::PoseTracking(const ros::NodeHandle& nh,
   initializePID(angular_pid_config_, cartesian_orientation_pids_);
 
   // Use the C++ interface that Servo provides
-  servo_ = std::make_unique<moveit_servo::Servo>(nh_, planning_scene_monitor_);
+  servo_ = std::make_unique<moveit_servo::Servo>(node_, parameters_, planning_scene_monitor_);
   servo_->start();
 
   // Connect to Servo ROS interfaces
-  target_pose_sub_ =
-      nh_.subscribe<geometry_msgs::PoseStamped>("target_pose", 1, &PoseTracking::targetPoseCallback, this);
+  target_pose_sub_ = node_->create_subscription<geometry_msgs::msg::PoseStamped>(
+     "target_pose", 1, std::bind(&PoseTracking::targetPoseCallback, this, std::placeholders::_1));
 
   // Publish outgoing twist commands to the Servo object
   twist_stamped_pub_ =
-      nh_.advertise<geometry_msgs::TwistStamped>(servo_->getParameters().cartesian_command_in_topic, 1);
+     node_->create_publisher<geometry_msgs::msg::TwistStamped>(servo_->getParameters()->cartesian_command_in_topic, 1);
 }
 
 PoseTrackingStatusCode PoseTracking::moveToPose(const Eigen::Vector3d& positional_tolerance,
                                                 const double angular_tolerance, const double target_pose_timeout)
 {
   // Wait a bit for a target pose message to arrive.
   // The target pose may get updated by new messages as the robot moves (in a callback function).
-  const ros::Time start_time = ros::Time::now();
+  const rclcpp::Time start_time = node_->now();
   while ((!haveRecentTargetPose(target_pose_timeout) || !haveRecentEndEffectorPose(target_pose_timeout)) &&
-         ((ros::Time::now() - start_time).toSec() < target_pose_timeout))
+         ((node_->now() - start_time).to_chrono<std::chrono::duration<double>>().count() < target_pose_timeout))
   {
     if (servo_->getCommandFrameTransform(command_frame_transform_))
     {
-      command_frame_transform_stamp_ = ros::Time::now();
+      command_frame_transform_stamp_ = node_->now();
     }
-    ros::Duration(0.001).sleep();
+    std::this_thread::sleep_for(std::chrono::milliseconds(1));
```

Comment:
How about using rclcpp::Duration with chrono literals?
```suggestion
    rclcpp::Duration(1ms).sleep();
```

Reply:
rclcpp::Duration doesn't seem to have a sleep function